libstuff/libstuff.h

```diff
@@ -246,14 +250,42 @@ extern atomic<void (*)(int priority, const char *format, ...)> SSyslogFunc;
     } while (false)
 
 #define SLOGPREFIX ""
-#define SDEBUG(_MSG_) SSYSLOG(LOG_DEBUG, "[dbug] " << SLOGPREFIX << _MSG_)
-#define SINFO(_MSG_) SSYSLOG(LOG_INFO, "[info] " << SLOGPREFIX << _MSG_)
-#define SHMMM(_MSG_) SSYSLOG(LOG_NOTICE, "[hmmm] " << SLOGPREFIX << _MSG_)
-#define SWARN(_MSG_) SSYSLOG(LOG_WARNING, "[warn] " << SLOGPREFIX << _MSG_)
-#define SALERT(_MSG_) SSYSLOG(LOG_ALERT, "[alrt] " << SLOGPREFIX << _MSG_)
+#define SDEBUG(_MSG_, ...)                          \
+    do {                                            \
+        stringstream ss;                            \
+        ss << "[dbug] " << SLOGPREFIX << _MSG_;     \
+        SSYSLOG(LOG_INFO, ss.str(), ##__VA_ARGS__); \
```

Comment:
if you use `fmt::format`, it will be a few times faster than using stringstream - https://mariusbancila.ro/blog/2023/09/12/formatting-text-in-c-the-old-and-the-new-ways